rxjava2/src/main/java/io/vertx/reactivex/package-info.java

```diff
@@ -0,0 +1,361 @@
+ * data.
+ *
+ * When you need to delay the subscription, you need to `pause` the `ReadStream` and then `resume` it, which is what
+ * you would do with a `ReadStream`.
+ *
+ * [source,java]
+ * ----
+ * {@link examples.RxifiedExamples#delayFlowable}
+ * ----
+ *
+ * Likewise it is possible to turn an existing `Observable` into a Vert.x `ReadStream`.
+ *
+ * The {@link io.vertx.reactivex.FlowableHelper#toReadStream(io.reactivex.Flowable)}  static methods convert
+ * a `Flowable` to a Vert.x read stream:
+ *
+ * [source,java]
+ * ----
+ * {@link examples.NativeExamples#toReadStream}
+ * ----
+ *
+ * === Async result support
+ *
+ * You can create an RxJava `Observer` from an existing Vert.x `Handler<AsyncResult<T>>` and subscribe
+ * it:
+ *
+ * [source,java]
+ * ----
+ * {@link examples.NativeExamples#handlerToSingleObserver}
+ * ----
+ *
+ * [source,java]
+ * ----
+ * {@link examples.NativeExamples#handlerToMaybeObserver}
+ * ----
+ *
+ * [source,java]
+ * ----
+ * {@link examples.NativeExamples#handlerToCompletableObserver}
+ * ----
+ *
+ * The _Rxified_ Vert.x API duplicates each such method with the `rx` prefix that returns an RxJava {@code Single},
+ * {@code Maybe} or {@code Completable}:
+ *
+ * [source,java]
+ * ----
+ * {@link examples.RxifiedExamples#single(io.vertx.reactivex.core.Vertx)}
+ * ----
+ *
+ * Such single are *cold* singles, and the corresponding API method is called on subscribe.
+ *
+ * `Maybe` can produce a result or no result:
+ *
+ * [source,java]
+ * ----
+ * {@link examples.RxifiedExamples#maybe}
+ * ----
+ *
+ * `Completable` is usually mapped to `Handler<AsyncResult<Void>>`
+ *
+ * [source,java]
+ * ----
+ * {@link examples.RxifiedExamples#completable}
+ * ----
+ *
+ * === Scheduler support
+ *
+ * The reactive extension sometimes needs to schedule actions, for instance `Flowable#timer` creates and returns
+ * a timer that emit periodic events. By default, scheduled actions are managed by RxJava, it means that the
+ * timer threads are not Vert.x threads and therefore not executing in a Vert.x event loop.
+ *
+ * When an RxJava method deals with a scheduler, it accepts an overloaded method accepting an extra `io.reactivex.Scheduler`,
+ * the {@link io.vertx.reactivex.RxHelper#scheduler(io.vertx.core.Vertx)}  method will return a scheduler that can be used
+ * in such places.
+ *
+ * [source,java]
+ * ----
+ * {@link examples.NativeExamples#scheduler(io.vertx.core.Vertx)}
+ * ----
+ *
+ * For blocking scheduled actions, a scheduler can be created with the {@link io.vertx.reactivex.RxHelper#blockingScheduler}
+ * method:
+ *
+ * [source,java]
+ * ----
+ * {@link examples.NativeExamples#blockingScheduler}
+ * ----
+ *
+ * RxJava can also be reconfigured to use the Vert.x scheduler:
+ *
+ * [source,java]
+ * ----
+ * {@link examples.NativeExamples#schedulerHook(io.vertx.core.Vertx)}
+ * ----
+ *
+ * NOTE: RxJava uses the words `computation schedule` for non-blocking tasks and `io schedule` for blocking tasks
```

Comment:
Do you mean `computation scheduleR` and `io scheduleR` ? Actually `computation` and `io` would be enough.